.pylintdict

```diff
@@ -509,6 +512,7 @@ parameterexpression
 parameterization
 parameterizations
 parameterizes
+parametrised
```

Comment:
Similar thing here - use the version in the source with the `z`  (that is already here) not the `s` variant

Reply:
Removed and changed the comments.